### PR TITLE
Optimize syntax for DB connection files

### LINKED ISSUE #74: Optimize DB connection file syntax
As MacGyver stated in #71, there are improvements that can be made to the DB connection files that can improve space and code efficiency.

### PR DESCRIPTION
**Pull request by salazw1**
Fixes #74 

As MacGyver stated in #71, the dates of birth should be stored in Unix timestamps, and try-catch block can be optimized by using a 'finally' statement.

- Converted the dates of births from YYYY-MM-DD string format to Unix timestamp for DB space efficiency
- Try-Catch block now includes a 'finally' block to ensure that the DB connection will close if any exception is caught during DB processing

### CODE REVIEW

File: small-talk/db/registerToDB.js [hunk 1]
```diff
@@ -3,26 +3,27 @@ const uri = "mongodb+srv://smt_root:pokemonwithguns@smalltalkcluster0.jo4jne6.mo
...
-    await mongoboi.disconnect();
     return newUser; // Returning the newly inserted user
   } 
-  
   catch (error) {
     console.error("Error writing to database:", error);
-    await mongoboi.disconnect();
     return null; // Returning null if there's an error
   }
+  finally {
```
Comment (aiharc1) [inline small-talk/db/registerToDB.js:26]: This is a really good addition. We overlooked disconnecting from our db

**Review by aiharc1 — APPROVED** [review]
Nice work Warrren. Good catch with the disconnecting from the db and changing to unix time to save space. Thanks again MacGyver for your help too.

I ran the app to confirm everything works as it should. Good job

**Review by jusasin — APPROVED** [review]
Seems like good work, if you could possibly add more documentation to make it easier to follow at first glance that would be really helpful. Good work on making sure to add a disconnect after executing the try or catch block.

File: small-talk/db/handleLogin.js [hunk 2]
```diff
@@ -17,18 +16,4 @@ export default async function handleSubmit(username, password) {
...
-    body: JSON.stringify({ email, password }),
-  })
-
-  if (response.ok) {
-    router.push('/profile')
-  } else {
-    // Handle errors
-  }
-  */
 }
```
Comment (blake-gilmore) [inline small-talk/db/handleLogin.js:19]: :)

**Review by blake-gilmore — APPROVED** [review]
Good, quick, and perfectly incremental changes.

File: small-talk/db/registerToDB.js [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
-    await mongoboi.disconnect();
     return newUser; // Returning the newly inserted user
   } 
-  
   catch (error) {
     console.error("Error writing to database:", error);
-    await mongoboi.disconnect();
     return null; // Returning null if there's an error
   }
+  finally { // ensure that the database connection is closed even if an exception is caught/thrown
```
Comment (junxjr) [inline small-talk/db/registerToDB.js:26]: great change, now that the finally block will always run to close the connection.

**Review by junxjr — APPROVED** [review]
The change looks good to me. It ensures the connection will always be closed by adding the finally block.

**Pull request merged**